spdb/project/resource.py

```diff
@@ -244,7 +244,7 @@ class Channel:
 
     """
     def __init__(self, name, description, ch_type, datatype, base_resolution, sources, related,
-                 default_time_sample, downsample_status):
+                 default_time_sample, downsample_status, storage_type, bucket, cv_path):
```

Comment:
What do you think about assigning defaults for these new variables? Then we _might_ still be compatible with older code.